documentation/book/appendix_metrics.adoc

```diff
@@ -37,21 +37,30 @@ endif::Kubernetes[]
 
 === Prometheus
 
-The provided Prometheus https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/prometheus.yaml[`prometheus.yaml`] YAML file describes all the resources required by Prometheus in order to effectively monitor a Strimzi Kafka & ZooKeeper cluster.
-These resources lack important production configuration to run a healthy and highly available Prometheus server.
-They should only be used to demonstrate this Grafana dashboard example.
+In Strimzi we use CoreOS Prometheus Operator which can be deployed by applying these https://github.com/coreos/prometheus-operator/tree/master/example/rbac/prometheus-operator[`Prometheus Operator YAML files`] from CoreOS repository.
+This operator allows us to run healthy and highly available Prometheus server what is one of the requirements for the production.
+
+If you use another namespace than in the repository files do you can download and edit single file by using a command.
+
+[source,shell,subs=attributes+]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-deployment.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/;s/regex: myproject/regex: _my-namespace_/' > prometheus-operator-deployment.yaml
```

Comment:
you are just downloading the YAML applying `sed` but then no oc/kubectl apply command is executed for really deploying the operator.